assets/js/modules/ads/components/settings/SettingsView.js

```diff
@@ -52,50 +55,62 @@ export default function SettingsView() {
 	return (
 		<Fragment>
 			<AdBlockerWarning />
-			{ ! isAdBlockerActive && ! paxConversionID && ! extCustomerID && (
-				<div className="googlesitekit-settings-module__meta-item">
-					<h5 className="googlesitekit-settings-module__meta-item-type">
-						{ __( 'Conversion Tracking ID', 'google-site-kit' ) }
-					</h5>
-					<p className="googlesitekit-settings-module__meta-item-data">
-						{ conversionID ? (
-							<DisplaySetting value={ conversionID } />
-						) : (
-							__( 'None', 'google-site-kit' )
-						) }
-					</p>
-				</div>
-			) }
+			{ ! isAdBlockerActive &&
+				( ( ! paxConversionID && ! extCustomerID ) ||
+					! paxEnabled ) && (
+					<div className="googlesitekit-settings-module__meta-item">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ __(
+								'Conversion Tracking ID',
+								'google-site-kit'
+							) }
+						</h5>
+						<p className="googlesitekit-settings-module__meta-item-data">
+							{ conversionID ? (
+								<DisplaySetting value={ conversionID } />
+							) : (
+								__( 'None', 'google-site-kit' )
+							) }
+						</p>
+					</div>
+				) }
 
-			{ ! isAdBlockerActive && paxConversionID && extCustomerID && (
-				<div className="googlesitekit-settings-module__meta-item">
-					<h5 className="googlesitekit-settings-module__meta-item-type">
-						{ __( 'Conversion Tracking ID', 'google-site-kit' ) }
-					</h5>
-					<p className="googlesitekit-settings-module__meta-item-data">
-						{ paxConversionID ? (
-							<DisplaySetting value={ paxConversionID } />
-						) : (
-							__( 'None', 'google-site-kit' )
-						) }
-					</p>
-				</div>
-			) }
+			{ ! isAdBlockerActive &&
+				paxEnabled &&
+				( paxConversionID || extCustomerID ) && (
+					<div className="googlesitekit-settings-module__meta-item">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ __(
+								'Conversion Tracking ID',
+								'google-site-kit'
+							) }
+						</h5>
+						<p className="googlesitekit-settings-module__meta-item-data">
+							{ paxConversionID ? (
+								<DisplaySetting value={ paxConversionID } />
+							) : (
+								__( 'None', 'google-site-kit' )
+							) }
+						</p>
+					</div>
+				) }
 
-			{ ! isAdBlockerActive && paxConversionID && extCustomerID && (
-				<div className="googlesitekit-settings-module__meta-item">
-					<h5 className="googlesitekit-settings-module__meta-item-type">
-						{ __( 'Customer ID', 'google-site-kit' ) }
-					</h5>
-					<p className="googlesitekit-settings-module__meta-item-data">
-						{ extCustomerID ? (
-							<DisplaySetting value={ extCustomerID } />
-						) : (
-							__( 'None', 'google-site-kit' )
-						) }
-					</p>
-				</div>
-			) }
+			{ ! isAdBlockerActive &&
+				paxEnabled &&
+				( paxConversionID || extCustomerID ) && (
```

Comment:
This conditional check is pretty crowded, making it a bit harder to follow. Perhaps we can make a new variable `shouldRenderExtCustomerID = paxEnabled && ( paxConversionID || extCustomerID )` and then use that single variable as only check at this point to determine if markup should be rendered.

Reply:
I've simplified this somewhat @zutigrm